inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/preprocessing.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multi-device/multi_device_config.hpp"
+
+#include "behavior/preprocessing.hpp"
+
+using namespace BehaviorTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
+    InferenceEngine::Precision::U16,
+    InferenceEngine::Precision::FP32
```

Comment:
Does it make sense to extend for other types as well?

Reply:
IMHO No. 
At least not for the test as it now. CPU plugin does internal precision conversion for other types as well, but only if there is mean image on input. Which is not handled by the proposed test.